tests/cases/compiler/checkJsdocTypeTagOnExportAssignment1.ts

```diff
@@ -0,0 +1,14 @@
+// @allowJs: true
+// @checkJs: true
+// @outDir: ./out
+// @filename: checkJsdocTypeTagOnExportAssignment1.js
+
+// @Filename: a.js
+/**
+ * @typedef {Object} Foo
+ * @property {boolean} a
+ * @property {boolean} b
+ */
+
+/** @type {Foo} */
```

Comment:
Please add another file that shows that the type of `import a from './a'` is `Foo` and not `{ c: boolean }`.